deepchem/models/torch_models/layers.py

```diff
@@ -2972,3 +2972,65 @@ def _dynamic_partition(self, input_tensor: torch.Tensor,
         partitions = [input_tensor[mask] for mask in partition_masks]
 
         return partitions
+
+
+class DTNNEmbedding(nn.Module):
+    """DTNNEmbedding layer for DTNN model.
+    Assign initial atomic descriptors.
+
+    Parameters
+    ----------
+    n_embedding: int, optional
+        Number of features for each atom
+    periodic_table_length: int, optional
+        Length of embedding, 83=Bi
+    init: str, optional
+        Weight initialization for filters.
```

Comment:
Need to specify what options are possible

Reply:
I think 'init' name is apt there. Can you suggest what i should change it to?
I have written the options available for init. I have skipped some of the options as there seems to be too many of them, do you want that i write all of them.